store/templates/store/store_list.html

```diff
@@ -3,13 +3,15 @@
 {% block content %}
         <main class="store-list">
             {% for store in stores %}
+                {% if store.image %}
```

Comment:
If there is no store image the store is not rendered even though there is a name and category.
- [ ] move the if block to the line that renders the image instead of the entire section

Reply:
I've decided to use an image with logic to handle images if one isn't supplied by default.